data/json/monsters/insect_spider.json

```diff
@@ -1581,7 +1581,7 @@
     "weight": "815 kg",
     "melee_dice": 2,
     "special_attacks": [ [ "SMASH", 30 ], [ "BIO_OP_TAKEDOWN", 20 ], [ "RANGED_PULL", 20 ], [ "GRAB", 5 ] ],
-    "extend": { "flags": [ "DESTROYS", "PUSH_MON", "PUSH_VEH" ] }
+    "extend": { "flags": [ "DESTROYS", "PUSH_MON", "PUSH_VEH", "RANGED_ATTACKER" ] }
```

Comment:
IDK why, but this is warning on load that it doesn't recognize the flag.

Reply:
Probably needs a json_flag definition.

Reply:
You mean on game load? Because I don't have any errors when I load a game.